coalib/collecting/Collectors.py

```diff
@@ -243,7 +243,8 @@ def collect_bears(bear_dirs, bear_globs, kinds, log_printer=None,
                              'No bears matching \'{}\' were found. Make sure '
                              'you have coala-bears installed or you have typed '
                              'the name correctly.')
-    return bears_found
+    return tuple(map(lambda x: sorted(x, key=lambda y: y.__name__),
```

Comment:
We can sort down at this level.
Too many things will change slightly, and perform badly, especially if a different sorting is needed higher up the stack.
Sorting needs to be done in the CLI layer where the desired use of the data is known.